recipes/spacy-legacy/meta.yaml

```
@@ -0,0 +1,50 @@
+{% set name = "spacy-legacy" %}
+{% set modulename = "spacy_legacy" %}
+{% set version = "3.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 3443becabcdcb797df95144c9865ab28f08e2e1a36653626e65f7c2a73ce7a08
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python
```

If (and only if!) it can be `noarch: python`:
```suggestion
    - python >=3.6
```